### PR TITLE
Feed experiment for 10-04-2022

### PR DESCRIPTION
**Pull request by benhalpern**
## What type of PR is this? (check all applicable)

- [ ] Refactor
- [ ] Feature
- [ ] Bug Fix
- [x] Optimization
- [ ] Documentation Update

## Description

Feed experiment outlined here:
https://docs.google.com/document/d/1H1WsktwvlspsB_1RmsZxfVcJHVcaeorHLjpAWG8Oazw

This week adds _two_ new variants and re-tests one from a few weeks ago.

- One experiment takes the winner and only flips `reseed_randomizer_on_each_request` to `true`
- The other experiment adds the following lever in _without replacing any existing levers_ (i.e. we now use count and point sum)

```
    "matching_positive_tags_intersection_count": {
      "cases": [
        [0, 0.55],
        [1, 0.98]
      ],
      "fallback": 1
    },
```

## Related Tickets & Documents

- Related Issue https://github.com/forem/forem/issues/18538
- Closes https://github.com/forem/forem/issues/18538

## QA Instructions, Screenshots, Recordings

_Please replace this line with instructions on how to test your changes, a note
on the devices and browsers this has been tested on, as well as any relevant
images for UI changes._

### UI accessibility concerns?

_If your PR includes UI changes, please replace this line with details on how
accessibility is impacted and tested. For more info, check out the
[Forem Accessibility Docs](https://developers.forem.com/frontend/accessibility)._

## Added/updated tests?

- [ ] Yes
- [x] No, and this is why: This is all already tested
- [ ] I need help with writing tests

## [Forem core team only] How will this change be communicated?

_Will this PR introduce a change that impacts Forem members or creators, the
development process, or any of our internal teams? If so, please note how you
will share this change with the people who need to know about it._

- [ ] I've updated the [Developer Docs](https://developers.forem.com) or
      [Storybook](https://storybook.forem.com/) (for Crayons components)
- [ ] This PR changes the Forem platform and our documentation needs to be
      updated. I have filled out the
      [Changes Requested](https://github.com/forem/admin-docs/issues/new?assignees=&labels=&template=changes_requested.md)
      issue template so Community Success can help update the Admin Docs
      appropriately.
- [ ] I've updated the README or added inline documentation
- [ ] I've added an entry to
      [`CHANGELOG.md`](https://github.com/forem/forem/tree/main/CHANGELOG.md)
- [ ] I will share this change in a [Changelog](https://forem.dev/t/changelog)
      or in a [forem.dev](http://forem.dev) post
- [x] I will share this change internally with the appropriate teams
- [ ] I'm not sure how best to communicate this change and need help
- [ ] This change does not need to be communicated, and this is why not: _please
      replace this line with details on why this change doesn't need to be
      shared_

### CODE REVIEW

File: config/feed-variants/20221004-variant-b.json [hunk 1]
```diff
@@ -0,0 +1,158 @@
+{
+  "max_days_since_published": 15,
+  "description": "Copy of 20221004-variant-b but with matching_positive_tags_intersection_count present in addition to the points sum",
```
Comment (fdocr) [inline config/feed-variants/20221004-variant-b.json:3]: Likely a typo that makes no meaningful difference, but this should be `Copy of 20221004-variant-a`, right?

**Review by fdocr — APPROVED** [review]
LGTM

File: config/feed-variants/20221004-variant-b.json (hunk 1 above)
Comment (benhalpern) [inline config/feed-variants/20221004-variant-b.json:3]:
```suggestion
  "description": "Copy of 20220926-variant-b but with matching_positive_tags_intersection_count present in addition to the points sum",
```

Comment (benhalpern) [reply config/feed-variants/20221004-variant-b.json:3]: Good catch

**Review by jaw6 — APPROVED** [review]

**Review by Ridhwana — APPROVED** [review]

**Pull request merged**